FlickrAssignment/NetworkManager .swift

```diff
@@ -43,7 +40,7 @@ class flickrNetworkClass: flickrNetwork {
                 self.data = decodedResponce.photos.photo
                print("data")
                 
-                self.viewController?.getFinalResult()
+                self.delegate?.updateImageResponce(responce: decodedResponce)
```

Comment:
instead of using protocol delegate pattern here , use  closure with Result type.